src/søknad/SendSøknad.tsx

```diff
@@ -36,10 +38,19 @@ const SendSøknad = () => {
         })
       );
   };
-
   return (
     <>
       <Side tittel={'Oppsummering'}>
+        {søknad.sivilstatus.begrunnelseForSøknad &&
+        (søknad.sivilstatus.begrunnelseForSøknad.verdi ===
+          tekster.nb['sivilstatus.svar.samlivsbruddAndre'] ||
+          søknad.sivilstatus.begrunnelseForSøknad.verdi ===
+            tekster.nb['sivilstatus.svar.samlivsbruddForeldre']) ? (
```

Comment:
Tanken er å senere har enums som skal fungere som nøkler for svarsalternativene. Da slipper du å sjekke mot tekstfilene her. :) 

Hvis du vil se hvordan spørsmål er er satt opp, kan du sjekke ut configsa. F.eks. MedlemsConfig som inneholder spørsmål og svarsalternativer for medlemsspørsmål.

Reply:
ja ikke sant, ser det er nok nice :D